src/ray/common/buffer.h

```diff
@@ -45,6 +45,28 @@ class LocalMemoryBuffer : public Buffer {
   size_t size_;
 };
 
+/// Accumulative buffer which takes ownership of the data.
+class AccumulativeBuffer : public Buffer {
+ public:
+  AccumulativeBuffer() {}
+
+  AccumulativeBuffer(uint8_t *data, size_t size) { Append(data, size); }
```

Comment:
Document this? especially the fact that the data will be copied.

Reply:
please see above